src/test/tests_common.h

```diff
@@ -45,6 +45,44 @@
 
 #include "crt_internal.h"
 
+int g_shutdown;
+
+static inline int drain_queue(crt_context_t ctx)
```

Comment:
this function should be removed from src/test/common.h. And since you are touching it, could you put the return type on a separate line?

Reply:
common.h is included by other tests (threaded_rpc.h) not converted to no pmix so I cannot remove it. I'm planning to delete common.h all together when all tests are converted.
Still want me to move the line or wait for next patch?

Reply:
next patch is fine.